packages/@glimmer/manager/lib/public/template.ts

```diff
@@ -1,33 +1,26 @@
 import type { TemplateFactory } from '@glimmer/interfaces';
 import { debugToString } from '@glimmer/debug-util';
+import { debugAssert } from '@glimmer/global-context';
```

Comment:
`debugAssert`s *do* show up in the build output. They're still nicer than explicit `import.mneta.env.DEV` checks.